tests/testutils/data_generator.py

```diff
@@ -78,14 +79,30 @@ class TestDataGenerator(object):
         'is_valid': True,
     }
 
+    DEFAULT_REPO_UPDATED_DATE = 1557676201
```

Comment:
maybe add a quick comment on what date that number is?

Reply:
Seems perfectly obvious but ok :p